src/Raven.Server/NotificationCenter/Handlers/DatabaseNotificationCenterHandler.cs

```diff
@@ -6,11 +6,54 @@
 using Raven.Server.Documents;
 using Raven.Server.NotificationCenter.Notifications;
 using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
 
 namespace Raven.Server.NotificationCenter.Handlers
 {
     public class DatabaseNotificationCenterHandler : DatabaseRequestHandler
     {
+        [RavenAction("/databases/*/notification-center/get", "GET", AuthorizationStatus.ValidUser, EndpointType.Read, SkipUsagesCount = true)]
+        public async Task GetNotifications()
+        {
+            var postponed = GetBoolValueQueryString("postponed", required: false) ?? true;
+            var type = GetStringQueryString("type", false);
+            var shouldFilter = type != null && Enum.TryParse(typeof(NotificationType), type, out _);
```

Comment:
If we cannot parse we should throw instead of silently ignoring that

Reply:
done